usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -57,6 +63,20 @@
         " tas_components ARRAY<STRING>",
         "column_names": list(AWARD_SEARCH_COLUMNS),
     },
+    "recipient_profile": {
+        "model": RecipientProfile,
+        "source_query": recipient_profile_load_sql_strings,
+        "source_database": None,
+        "source_table": None,
+        "destination_database": "rpt",
+        "swap_table": "recipient_profile",
+        "swap_schema": "rpt",
+        "partition_column": "recipient_hash",  # This isn't used for anything
+        "partition_column_type": "string",
```

Comment:
Please add the new column underneath: 
`"is_partition_column_unique": False,`

Reply:
added